packages/svelte/src/lib/types.ts

```diff
@@ -0,0 +1,99 @@
+import type { ExtendObjectIf, UploadThingError } from "@uploadthing/shared";
+import type {
+  ClientUploadedFileData,
+  FileRouter,
+  inferEndpointInput,
+  inferEndpointOutput,
+  inferErrorShape,
+} from "uploadthing/types";
+
+export interface GenerateTypedHelpersOptions {
+  /**
+   * URL to the UploadThing API endpoint
+   * @example "/api/uploadthing"
+   * @example "https://www.example.com/api/uploadthing"
+   *
+   * If relative, host will be inferred from either the `VERCEL_URL` environment variable or `window.location.origin`
+   *
+   * @default (VERCEL_URL ?? window.location.origin) + "/api/uploadthing"
+   */
+  url?: string | URL;
+}
+
+export type UseUploadthingProps<
```

Comment:
Nit: should name match `createUploadthing`?

Reply:
Yes mb